sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -1217,13 +1217,13 @@ void ProgramManager::addImages(pi_device_binaries DeviceBinary) {
           // The supplied device_global info property is expected to contain:
           // * 8 bytes - Size of the property.
           // * 4 bytes - Size of the underlying type in the device_global.
-          // * 1 byte  - 0 if device_global has device_image_scope and any value
+          // * 4 bytes - 0 if device_global has device_image_scope and any value
           //             otherwise.
-          // Note: Property may be padded.
           assert(DeviceGlobalInfo.size() >= 13 && "Unexpected property size");
```

Comment:
```suggestion
          assert(DeviceGlobalInfo.size() == 16 && "Unexpected property size");
```

Reply:
Thank you @steffenlarsen this definitely makes sense.